Arch Linux compile error

The following compile error was reported on Arch Linux (via Discord in #ophd):
> NAS2D/Renderer/../Utility.h:58:28: error: expected unqualified-id before ‘)’ token
   58 |                 Utility<T>() = delete;

--- a/NAS2D/Utility.h
+++ b/NAS2D/Utility.h
@@ -55,10 +55,10 @@ namespace NAS2D
 	class Utility
 	{
 	public:
-		Utility<T>() = delete;
+		Utility() = delete;
 		~Utility() = delete;
-		Utility<T>(const Utility& s) = delete;
-		Utility<T>& operator=(const Utility& s) = delete;
+		Utility(const Utility& s) = delete;
+		Utility& operator=(const Utility& s) = delete;
 
 		/**
 		 * Gets a reference to a global instance of the specified type

